benchflow-experiment-manager/application/src/main/java/cloud/benchflow/experimentmanager/resources/BenchFlowExperimentResource.java

```diff
@@ -151,4 +152,21 @@ public BenchFlowExperimentStateResponse changeExperimentState(
 
     return new BenchFlowExperimentStateResponse(state);
   }
+
+  @POST
+  @Path("/abort")
+  public void abortExperiment(@PathParam("username") String username,
+      @PathParam("testName") String testName, @PathParam("testNumber") int testNumber,
+      @PathParam("experimentNumber") int experimentNumber) {
+
+    String experimentID =
+        BenchFlowConstants.getExperimentID(username, testName, testNumber, experimentNumber);
+
+    logger.info("request received: POST " + BenchFlowConstants.getPathFromExperimentID(experimentID)
+        + ABORT_PATH);
+
+    // execute in separate thread so we can return
```

Comment:
We said for now that it is fine to just wait for Faban to finish if something is running and then in the thesis I can write about more sophisticated methods. The experiment will then be aborted and the TM informed as described in https://github.com/benchflow/benchflow/issues/499